sandbox/package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Ross Bulat",
   "description": "Core UI Components for Polkadot Dashboards.",
   "scripts": {
+    "clear": "rm -rf node_modules",
```

Comment:
Will the top-level `clear` command cover the sandbox?

Reply:
Top level commands cover everything that exist in the workspaces array unless commanded differently.
If --workspace <name> is used then command will execute only for that workspace. Its better when --workspaces is used to add the --if-present in order to avoid errors.on missing commands from a workspace.